website/landingpage/dev/src/helpers/blogentries.js

```diff
@@ -1,4 +1,33 @@
 import { getBlogEntries } from '../services/blogprocessor';
-module.exports = (options) => {
-  return getBlogEntries().map(entry => entry.blogExcerpt).join(' ');
+import path from 'path';
+import { writeFileSync } from 'fs';
+
+module.exports = () => {
+  const blogChunkPath = path.join(__dirname, '../../../public/blog', 'blog-chunks');
+
+  const numberOfBlogsToBeVisibleOnLoad = 3,
```

Comment:
As discussed, I would update it now and leave a comment in the PR to lower the number for simpler testing.
```suggestion
  const numberOfBlogsToBeVisibleOnLoad = 5,
```